src/layout-grid/styled-components.js

```diff
@@ -14,39 +14,41 @@ import type {ResponsiveT, StyledGridPropsT, StyledCellPropsT} from './types.js';
 export const StyledGrid = styled<StyledGridPropsT>(
   'div',
   ({
+    $theme,
     $align = null,
     $behavior = BEHAVIOR.fixed,
-    $gridGutters = null,
-    $gridMargins = null,
-    $gridMaxWidth = null,
-    $theme,
+    $gridGutters = $theme.grid.gutters,
+    $gridMargins = $theme.grid.margins,
+    $gridMaxWidth = $theme.grid.maxWidth,
+    $unit = $theme.grid.unit,
```

Comment:
Currently all of the props that have defaults in the theme have a `grid*` prefix in the props for the styled components (`StyledGrid`, `StyledCell`). It's just convention but would be nice to preserve that.

```
theme       styled component
unit     -> $gridUnit
columns  -> $gridColumns
gaps     -> $gridGaps
margins  -> $gridMargins
gutters  -> $gridGutters
maxWidth -> $gridMaxWidth
```

Reply:
sure will make this change!

Reply:
done 👍